src/pages/Nav.js

```diff
@@ -0,0 +1,63 @@
+import React from "react";
+import "../css/nav.css";
+
+import { Route, Routes, Link } from "react-router-dom";
+import Home from "./Home";
+import Docs from "./Docs";
+import Label from "./Label";
+import Login from "./Login";
+import Notes from "./Notes";
+import Archive from "./Archive";
+import Trash from "./Trash";
```

Comment:
import React is no longer needed also for the components, you can do a barrel export to make the component clean!